From 7150045481c47ca1817f97c3593444a619e7419d Mon Sep 17 00:00:00 2001
From: KokaKiwi <kokakiwi+git@kokakiwi.net>
Date: Tue, 13 Feb 2024 00:04:04 +0100
Subject: [PATCH] makepkg: Disable faulty test

Got a compile error for utils_test:
```
mold: error: undefined symbol: testing::internal::PrintU8StringTo(std::__cxx11::basic_string<char8_t, std::char_traits<char8_t>, std::allocator<char8_t> > const&, std::ostream*)
>>> referenced by utils_test.cpp
>>>               CMakeFiles/utils_test.dir/test/utils_test.cpp.o:(testing::internal::UniversalTersePrinter<char8_t const*>::Print(char8_t const*, std::ostream*))>>> referenced by utils_test.cpp
```
---
 test/utils_test.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/utils_test.cpp b/test/utils_test.cpp
index 3537f0a..8f8d887 100644
--- a/test/utils_test.cpp
+++ b/test/utils_test.cpp
@@ -48,6 +48,7 @@ TEST(utils, utf8_display_width) {
                     u8"unicode/我爱你/☀️ Sun/Γειά σας/مرحبًا/⚽️/Карибського")));
 }
 
+#if 0
 TEST(utils, uft8_truncate) {
   auto u8trunc = [](std::u8string str, size_t len) {
     auto tmp = u8string_to_string(str);
@@ -75,6 +76,7 @@ TEST(utils, uft8_truncate) {
   EXPECT_EQ(u8"我", u8trunc(str, 2));
   EXPECT_EQ(u8"", u8trunc(str, 1));
 }
+#endif
 
 TEST(utils, shorten_path_ascii) {
   std::string const orig =
-- 
2.44.0

